openfisca_france/model/prelevements_obligatoires/droits_de_donation.py

```diff
@@ -0,0 +1,94 @@
+from openfisca_france.model.base import *
+
+
+class LienParente(Enum):
+    __order__ = 'aucun quatrieme_degre neveu fratrie arriere_petit_enfant petit_enfant enfant ascendant epoux_pacs'
+    aucun = 'Aucun lien de parenté'
+    quatrieme_degre = 'Parent de 4ème degré'
+    neveu = 'Neveu ou Nièce'
+    fratrie = 'Frère ou soeur'
+    arriere_petit_enfant = 'Arrière petit enfant'
+    petit_enfant = 'Petit enfant'
+    enfant = 'Enfant'
+    ascendant = 'Mère, père, grand-mère, grand-père'
+    epoux_pacs = 'Epoux-se ou partenaire Pacs'
+
+
+class lien_parente(Variable):
+    value_type = Enum
+    possible_values = LienParente
+    default_value = LienParente.aucun
+    entity = Individu
+    label = 'Lien de parenté entre le donateur et le donataire'
+    definition_period = YEAR
+
+
+class droit_exoneration_familial(Variable):
+    value_type = bool
+    default_value = True
+    entity = Individu
+    label = 'Droit à l exonération familial, True si le donateur a moins de 80 ans et le donataire plus de 18 ans'
+    definition_period = YEAR
+
+
+class don(Variable):
+    value_type = float
+    default_value = 0.0
+    entity = Individu
+    label = 'Montant de donation'
+    definition_period = YEAR
+    reference = 'https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000030061736'
+
+
+class droits_mutation(Variable):
+    value_type = float
+    default_value = 0.0
+    entity = Individu
+    label = 'Droits de mutation à titre gratuit'
+    definition_period = YEAR
+    reference = 'https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000030061736'
```

Comment:
Utiliser des références datées pour éviter les changements de contenu et incompréhension d'un futur contributeur ?
```suggestion
    reference = 'https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000030061736/2014-12-30/'
```